python/notebooks/getting-started-gym.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
Since the getting-started-gym.ipynb is specific to the gym-unity folder, does it make more sense to move it under the gym-unity folder? 

Also another way is to move the gym-unity folder under python folder. 

Also within the getting-started-gym.ipynb and getting-started.ipynb, the default env_name is pointed to "../envs/3DBall" and "../envs/GridWorld", and in the python/notebooks/ folder, currently its parent folder doesn't contain any envs folder, which makes it confusing.

Reply:
I think in the re-org we will be doing, we will move the notebooks to the top level. 

I will be more specific about the need for an `envs` folder.